cli/dts/lib.deno.ns.d.ts

```diff
@@ -1982,13 +1982,12 @@ declare namespace Deno {
     stderrOutput(): Promise<Uint8Array>;
     close(): void;
 
-    /** **UNSTABLE**: The `signo` argument may change to require the Deno.Signal
-     * enum.
+    /** **UNSTABLE**
      *
      * Send a signal to process. This functionality currently only works on
      * Linux and Mac OS.
      */
-    kill(signo: number): void;
+    kill(signo: string): void;
```

Comment:
This also should be `Signal` type?

Reply:
I'm not sure how to do this because Signal type is unstable API... I think we should just allow less precise types here until this is made stable. I will add a TODO.

Reply:
On windows, wouldn't this just map to the status code? so should probably string or integer?

Reply:
@caspervonb Hmm that's a good question. @piscisaureus do you have any recommendations?

Reply:
> @caspervonb Hmm that's a good question. @piscisaureus do you have any recommendations?

* It's unusual for signal numbers to be mapped directly to exit codes on Windows.
* We want Deno to do something reasonable on all platforms even if they're not writing platform specific code. `p.kill("SIGKILL")` should work on Windows, and as is customary, it should exit the process with exit code 1.